cms-common/src/main/java/io/commercetools/sunrise/cms/CmsIdentifier.java

```diff
@@ -0,0 +1,74 @@
+package io.commercetools.sunrise.cms;
+
+import javax.annotation.Nullable;
+
+import static org.apache.commons.lang3.StringUtils.split;
+
+/**
+ * CMS Identifier, consisting of the field name and the content type and key.
+ *
+ * - {@code entryType}: type of entry which usually defines the fields it can contain (e.g. banner)
+ * - {@code entryKey}: key that identifies the particular entry (e.g. homeTopLeft)
+ * - {@code fieldName}: field name of the required content (e.g. subtitle)
+ *
+ * @see CmsService
+ */
+public final class CmsIdentifier {
```

Comment:
Actually it was extending Base, but in here I didn't put the dependency. Anyway this part it's just a rough copy of what I have in Sunrise, don't mind it that much.